src/lib/isMobilePhone.js

```diff
@@ -34,7 +34,7 @@ const phones = {
   'el-GR': /^(\+?30|0)?(69\d{8})$/,
   'en-AU': /^(\+?61|0)4\d{8}$/,
   'en-BM': /^(\+?1)?441(((3|7)\d{6}$)|(5[0-3][0-9]\d{4}$)|(59\d{5}))/,
-  'en-GB': /^(\+?44|0)7\d{9}$/,
+  'en-GB': /^(\+?44|0)7[1-9]\d{8}$/,
```

Comment:
@tux-tn and @profnandaa, do we need need to allow pagers and personal numbering as part of the phone numbers?
<img width="1265" alt="Screenshot 2022-06-03 at 04 57 32" src="https://user-images.githubusercontent.com/48981919/171772004-42b70a01-d5f9-4270-b773-52e75be0a916.png">

Reply:
what is personal numbering? Are people still using pagers these days?
![image](https://user-images.githubusercontent.com/261265/180596444-f867fd4b-af43-4718-83d9-2cc0648e1ebb.png)